cmd/registry/controller/local-tasks.go

```diff
@@ -36,12 +37,14 @@ func (task *ExecCommandTask) Run(ctx context.Context) error {
 		return fmt.Errorf("'resolve' not allowed in action, cannot invoke %q", task.Action)
 	}
 	cmd := exec.Command("registry", strings.Fields(task.Action)...)
-	output, err := cmd.CombinedOutput()
+	cmd.Stdout, cmd.Stderr = os.Stdout, os.Stderr
+	err := cmd.Run()
+	details := fmt.Sprintf("Finished execution %%s action={%s} taskID={%s}", task.Action, task.TaskID)
 
 	if err != nil {
+		log.Printf(details, "FAILED")
 		return err
 	}
-	log.Printf("Finished executing taskId %s with action: %s\n Output: %s", task.TaskID, task.Action, output)
-
+	log.Printf(details, "SUCCESS")
```

Comment:
Using the escaped formatter from the details variable is a bit confusing here. Could we capture less information in the details variable? These log messages can be `log.Printf("Failed execution of %s: %s", details, err)` and `log.Printf("Successful execution of %s", details)` if `details` only include the action and task ID.

Alternatively I actually think it's clearer to simply have two fully separate log messages that just repeat the common details, but there's no right answer for that.

Reply:
Right, thanks for calling it out. I ended up over optimizing the code. Made changes, hope it's more clear now